_episodes_rmd/01-introduction.Rmd

```diff
@@ -416,12 +416,15 @@ Error in .helpForCall(topicExpr, parent.frame()) :
 ~~~
 {: .error}
 
-
-Use two question marks (i.e. `?? geom_point()`) and R will return online search results in the "Help" tab. Finally, if you think there should be a function, for example a statistical test, but you aren't sure what R calls it, or what functions may be available, use the `help.search()` function.
+Use two question marks (i.e. `??geom_point()`) and R will return
+online search results in the "Help" tab. Finally, if you think there
```

Comment:
-> results from a search of the documentation for packages you have installed on your computer

It's not technically doing an online search. `??` and `help.search()` are synonyms, more details [here](https://www.r-project.org/help.html).